packages/office-ui-fabric-react/src/common/sharedIsConformant.tsx

```diff
@@ -0,0 +1,12 @@
+import { isConformant, IsConformantOptions } from '@fluentui/react-conformance';
+import { Ref, RefFindNode } from '@fluentui/react-component-ref';
+
+export function sharedIsConformant(testInfo: IsConformantOptions) {
+  const { disabledTests = [], wrapperComponents = [] } = testInfo;
+
+  isConformant({
```

Comment:
For correct deep merging of arrays, you need to use `merge` from `../Utilities`, and pass in the user options second. `merge(/*defaults*/, testInfo)`